Code/Framework/AzToolsFramework/AzToolsFramework/UI/PropertyEditor/GenericComboBoxCtrl.inl

```diff
@@ -202,6 +216,39 @@ namespace AzToolsFramework
         }
     }
 
+    template<typename T>
+    QComboBox* GenericComboBoxCtrl<T>::GetComboBox()
+    {
+        return m_pComboBox;
+    }
+
+    template<typename T>
+    QToolButton* GenericComboBoxCtrl<T>::GetEditButton()
+    {
+        return m_editButton;
+    }
+
+    template<typename T>
+    void GenericComboBoxCtrl<T>::SetEditButtonCallBack(GenericEditButtonCallback<T> function)
+    {
+        m_editButtonCallback = function;
+    }
+
+    template<typename T>
+    void GenericComboBoxCtrl<T>::OnEditButtonClicked()
+    {
+        if (m_editButtonCallback)
+        {
+            // A successful outcome returns a bool that determines the new value of the comboBox
```

Comment:
The comment is not quite correct.
The successful outcome in this case returns the new value to set